packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -185,21 +180,51 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
       clone.normalMap = material.normalMap.clone();
       clone.normalMap.needsUpdate = true;
     }
-    if (material.aoMap != null) {
-      clone.aoMap = material.aoMap.clone();
-      clone.aoMap.needsUpdate = true;
-    }
     if (material.emissiveMap != null) {
       clone.emissiveMap = material.emissiveMap.clone();
       clone.emissiveMap.needsUpdate = true;
     }
-    if (material.metalnessMap != null) {
+
+    // Clones the roughnessMap if it exists.
+    let roughnessMap: Texture|null = null;
+    if (material.roughnessMap) {
+      roughnessMap = material.roughnessMap!.clone();
```

Comment:
TS is smart enough that you don't need the `!` here. Also, we generally prefer to be more explicit with `material.roughnessMap != null`.

Reply:
Done.